lib/split/alternative.rb

```diff
@@ -43,20 +50,21 @@ def participant_count=(count)
 
     def completed_count(goal = nil)
       unless @completed_count
-        @completed_count = HashWithIndifferentAccess.new
-        self.goals.each do |goal|
+        @completed_count = ActiveSupport::HashWithIndifferentAccess.new
+        # add nil so we include overall completed count
+        (self.goals + [nil]).each do |goal|
```

Comment:
don't you want an empty array instead of an array that contains one nil element?